test/runtests.jl

```diff
@@ -12,6 +14,13 @@ for a in todownload
     download(repo*"/"*a, joinpath(tsfolder, a))
 end
 
+#New:
+repo = "https://raw.githubusercontent.com/JuliaDynamics/JuliaDynamics/master/timeseries"
+a = "test_time_series_lorenz_standard_N_10000_multivariate.csv"
+download(repo*"/"*a, joinpath(tsfolder, a))
+a = "test_time_series_roessler_N_10000_multivariate.csv"
+download(repo*"/"*a, joinpath(tsfolder, a))
+
```

Comment:
Should be written in terms of a loop. Define `files = [all file names here]` and loop over `for a in files`